Scripts/build-ios-universal.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 # Helper variables
+WORK_DIR=build
```

Comment:
Why do you need to introduce this folder?

Reply:
Why do you need to modify this script it all? No need to make another copy of frameworks, just use it from where it's placed